README.md

```diff
@@ -209,15 +219,18 @@ To send wrapped Bitcoin back to Bitcoin, run the following commands:
     ```
     axelarcli tx bitcoin sign-pending-transfers 0.0001btc --gas="auto" --gas-adjustment=1.15 --from validator -b block -y
     ```
-5. Submit the transfer to Bitcoin
+
+    **Note**: If you are having gas error issue `out of gas: out of gas in location: WriteFlat; gasWanted: 200000, gasUsed: 200780"`, try to increment the `gas-adjustement` to a higher value `1.2` or `1.3`. With more deposits, the transaction become larger and hence requires more gas
+
+3. Submit the transfer to Bitcoin
     ```
     axelarcli q bitcoin send
     -> returns tx ID
     ```
     You can monitor the status of your transfer using the bitcoin testnet explorer: https://blockstream.info/testnet/ .
 
 #### 🛑 **IMPORTANT: Verify outpoints of previous withdrawal tx (repeat for each outpoint)**
-Without this step, other users of the testnet will be unable to withdraw their wrapped tokens. Be a good citizen and verify the outpoints!
+    Without this step, other users of the testnet will be unable to withdraw their wrapped tokens. Be a good citizen and verify the outpoints!
```

Comment:
should we clarify what an `outpoint` is? i saw some folks were confused.

Reply:
yes i think its best to clarify

Reply:
@sergeynog yes agree, if you provide the definition here, i can update the PR, I am also not clear here, just guesses based on the verification that was required.

Reply:
@sergeynog I added a definition of a bitcoin outpoint

Reply:
how about this: replace "Bitcoin identifies transactions using a hash and an outpoint is a hash and index pair in which the index simply identifies which output a given transaction is intended." -> "Every Bitcoin transaction can have multiple outputs. Similarly, on your withdrawal transactions you will see multiple outputs even if you processed one withdrawal (the other outputs are used for internal controller functions). Please verify all outputs (i.e., {transaction hash, index} pairs) in your withdrawal transaction.